index.Rmd

```diff
@@ -0,0 +1,181 @@
+---
+title: "The Mother's Age and Birth Rates in the US"
+author: "Helen Chamley"
+date: '2022-05-09'
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE, fig.align = 'center')
+```
+
+```{r}
+source(here::here("scripts", "functions.R"))
+source(here::here("scripts", "get_data.R"))
+```
+
+```{r}
+births_clean <-
+  births %>% 
+    slice(-c(2,4:55, 57:108, 110:161, 163:214, 216:267, 
+             269:320, 322:373, 375:427, 429:480, 482:533, 
+             535:586, 588:640, 643, 646:698, 700:751, 
+             753:805, 806:858, 860:911, 913:964, 966:1017, 
+             1019:1070, 1072:1176, 1178:1229, 
+             1231:1282, 1284:1391)) %>% 
+  select(-c('1980', '1981', '1982', '1983', '1984', '1985', '1986', '1987', '1988', '2020')) 
+
+births_wide <-
+  births_clean %>%
+  pivot_longer(!Years, names_to = "year", values_to = "children_born") %>% 
+  rename(Type = Years, Year = year, Births = children_born) %>% 
+  select(Year, Births, Type) %>% 
+  mutate(Year = as.numeric(Year)) %>%
+  arrange(Year) %>%
+  mutate(Type = str_replace(string = Type,
+                            pattern = "Less than 15",
+                            replacement = "15 and younger")) %>% 
+  mutate(Type = str_replace(string = Type,
```

Comment:
If all of them have the same prefix, do you actually need the prefix? Would it actually make sense to remove it?

Reply:
Nevermind - I see that you took care of that later on!